README.md

```diff
@@ -221,6 +221,19 @@ variable definition in the vars folder.
 
 Enables / disables os package installation.
 
+    controlhost_terraform_utilities
+    
+Enables / disables Terraform convenience utilities setup, including Bash aliases and setting `TF_DATA_DIR`.
+This also allows for using a non-rsync folder sync with Windows hosts due to Terraform symlinks being created outside of
+the sync scope.
+
+    # controlhost_terraform_project_name: sample-project
```

Comment:
we should name this parameter controlhost_terraform_data_dir. We can then set this in the archetype to the project name. But the controlhost does not know or referenc to any project.

Reply:
✔️